spinnaker_camera_driver/cmake/download_spinnaker

```diff
@@ -87,32 +183,16 @@ if not os.path.exists(os.path.join(os.getcwd(), "usr/lib/")):
     logging.info("Unpacking debs.")
     debs = glob.glob(os.path.join(os.getcwd(), folder_name, "libspinvideoencoder-*.deb"))
     debs += glob.glob(os.path.join(os.getcwd(), folder_name, "*spinnaker-*.deb"))
+    unpack_dir = os.path.join(os.getcwd(), 'flir_camera_driver', 'spinnaker_camera_driver')
```

Comment:
Need to create this folder if it doesn't exist.

Reply:
When I build the packages with catkin_make I don't get any errors related to the directory not existing.  I can add an explicit check & make the directories manually, but it feels like its not necessary?

Reply:
Fixed.